lib/ts/controllers/s-expandable-control.ts

```diff
@@ -79,13 +79,13 @@
         };
 
 
-        get controlledCollapsible() {
+        get controlledCollapsibles() {
             const attr = this.element.getAttribute("aria-controls");
             if (!attr) {
                 throw "couldn't find controls"
```

Comment:
While we're in here making changes, I'm going to change this error message a tad.

```suggestion
                throw `[aria-controls="targetId1 ... targetIdN"] attribute required`;
```